.golangci.yml

```diff
@@ -48,6 +49,14 @@ linters-settings:
       - pkg: "github.com/opencontainers/go-digest"
         alias: "digest"
     no-unaliased: true
+  gosec:
+    excludes:
+      - G101  # Potential hardcoded credentials (false positives)
+      - G402  # TLS MinVersion too low
+      - G601  # Implicit memory aliasing in for loop (false positives)
+      - G504  # Import blocklist: net/http/cgi
```

Comment:
These comments could be improved. Eg. why do we have an import on "net/http/cgi" and what do we need to do to fix it.

Reply:
We only use it for testing in [gitsource_test.go](https://github.com/moby/buildkit/blob/master/source/git/gitsource_test.go#L608-L626) to use `git http-backend` - we could just remove the warning in that file, but the gosec warning itself is very weird.

The only reason it errors is that there's a vulnerability in an old version of go... that we don't support. I think it's completely reasonable to disable it globally, since we don't target that version.